package.json

```diff
@@ -33,8 +33,8 @@
   "homepage": "https://github.com/moneyforward/eslint-config-moneyforward#readme",
   "dependencies": {
     "@next/eslint-plugin-next": "^14.1.3",
-    "@typescript-eslint/eslint-plugin": "^6.21.0",
-    "@typescript-eslint/parser": "^6.21.0",
+    "@typescript-eslint/eslint-plugin": "^7.0.0",
+    "@typescript-eslint/parser": "^7.0.0",
```

Comment:
Each package has been released as `v7.10.0`, how about update to the latest version?

https://www.npmjs.com/package/@typescript-eslint/eslint-plugin
https://www.npmjs.com/package/@typescript-eslint/parser

Reply:
fix: [1802efc](https://github.com/moneyforward/eslint-config-moneyforward/pull/273/commits/1802efce7151f8a87bc4470b4688fd7efda4ae21)